tools.py

```diff
@@ -516,12 +515,12 @@ def split(
 	)
 
 
-def cleanup_transcripts_with_empty_ref(input_path, emptiness):
+def cleanup_transcripts_with_empty_ref(input_path, min_ref_length):
 	transcript_array = json.load(open(input_path))
 
 	filtered = []
 	for transcript in transcript_array:
-		if len(transcript.get('ref', "")) <= emptiness:
+		if len(transcript.get('ref', "")) <= min_ref_length:
```

Comment:
`min_ref_length` should be inclusive. please change the comparison to ` < min_ref_length`

Reply:
fixed